.github/workflows/release.yaml

```
@@ -0,0 +1,147 @@
+
+    - name: Checkout BMF repository (specific branch)
+      run: |
+        sudo apt update
+        sudo apt install -y make git pkg-config libssl-dev cmake binutils-dev libgoogle-glog-dev gcc g++ golang wget libgl1
+        sudo apt install -y python3.9 python3-dev python3-pip libsndfile1 libsndfile1-dev
+        # sudo python3 -m pip install timeout_decorator numpy onnxruntime pytest opencv-python librosa
+        
+        git clone https://github.com/JackLau1222/bmf.git
+
+    - name: Cache FFmpeg build
+      uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)